src/main/java/com/wultra/app/enrollmentserver/database/entity/OperationTemplateParam.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * PowerAuth Enrollment Server
+ * Copyright (C) 2021 Wultra s.r.o.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package com.wultra.app.enrollmentserver.database.entity;
+
+import lombok.Data;
+
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Class representing an operation template attribute.
+ *
+ * @author Petr Dvorak, petr@wultra.com
+ */
+@Data
+public class OperationTemplateParam implements Serializable {
+
+    private String type;
+    private String id;
+    private String text;
+    private final Map<String, String> params = new HashMap<>();
```

Comment:
Isn't it better to use a `LinkedHashMap` to keep the same order of parameters during serialization?